### PR TITLE
Feature/fmfr 220 service volume import

### PR DESCRIPTION
**Pull request by ramchurnk**
logic for service volumes and lifts addition

fixed specs

mock spreadsheet sheets created for service volumes

Refactor spec and helper

Upgrade template to v2.6

Add logging around basic upload file validations

Remove redundant before blocks

Fix failing specs around new template version

Add service matrix to fake spreadsheet

WIP service matrix spec

Extract the service codes and standards from the matrix

initial working spec

lift and service hour specs with improved importer

fix failing specs

fixed other issues in bundle

### CODE REVIEW

File: app/services/facilities_management/spreadsheet_importer.rb [hunk 1]
```diff
@@ -204,9 +190,69 @@ def validate_procurement_building(procurement_building_hash, building)
...
+      end
+    else
+      @errors << :lifts_incomplete
+    end
+  end
 
+  def get_lift_data(service_volume_column)
+    lifts = []
+
+    40.times do |row|
```
Comment (doctorpod) [inline app/services/facilities_management/spreadsheet_importer.rb:245]: Make this a static e.g. NUM_LIFTS = 40

File: spec/support/spreadsheet_import_helper.rb [hunk 2]
```diff
@@ -60,6 +60,51 @@ def add_service_matrix_sheet(data)
...
+      name = 'Service Volumes 2'
+      @sheets_added << name
+      @package.workbook.add_worksheet(name: name) do |sheet|
+        sheet.add_row(['', '', '', '', 'Service status indicator'] + service_details.map { |pb| pb[2] })
+        sheet.add_row(['', '', '', '', 'Building name'] + service_details.map { |pb| pb[0] })
+        sheet.add_row(['', '', '', '', 'Service required within this building?'] + service_details.map { |pb| pb[1].any? ? 'Yes' : 'No' })
+        sheet.add_row(['', '', '', '', 'Number of lifts in each building'] + service_details.map { |pb| pb[1].length })
+        sheet.add_row([])
+        sheet.add_row(['Service Reference', 'Service Name', 'Service required within this estate?', 'Lift Number', 'Metric'] + service_details.map { |pb| pb[0] })
+        40.times do |i|
```
Comment (doctorpod) [inline spec/support/spreadsheet_import_helper.rb:98]: Use the static (see above)

File: app/services/facilities_management/spreadsheet_importer.rb (hunk 1 above)
Comment (tim-s-ccs) [reply app/services/facilities_management/spreadsheet_importer.rb]: This has now been added

File: spec/support/spreadsheet_import_helper.rb (hunk 2 above)
Comment (tim-s-ccs) [reply spec/support/spreadsheet_import_helper.rb]: This has now been added

**Pull request merged**